styles/modernist.css

```
@@ -90,24 +98,31 @@ May 2013
   float: right;
   margin-right: 2.2321429%; /* 20/896px */
   min-height: 12em;
-  width: 18.3035714%; /* 164/896px */
   position: relative;
+  width: 18.3035714%; /* 164/896px */
 }
+
+/*Push the main body down to counteract the absolute positioning */
```

Inconsequential change, just changing for consistency
```suggestion
/* Push the main body down to counteract the absolute positioning */
```